Gui/ColorSelectorWidget.cpp

```diff
@@ -454,8 +646,20 @@ ColorSelectorWidget::ColorSelectorWidget(bool withAlpha, QWidget *parent)
                          this, SLOT( handleSliderAMoved(double) ) );
     }
 
-    QObject::connect( _button, SIGNAL( clicked(bool) ),
-                      this, SLOT( handleButtonClicked(bool) ) );
+    QObject::connect( _buttonColorGroup, SIGNAL( buttonClicked(int) ),
+                      this, SLOT( handleButtonColorClicked(int) ) );
+
+    QObject::connect( paletteAddColorButton, SIGNAL( clicked(bool) ),
+                      this, SLOT( setPaletteButtonColor(bool) ) );
+    QObject::connect( paletteClearButton, SIGNAL( clicked(bool) ),
+                      this, SLOT( clearPaletteButtons(bool) ) );
+    QObject::connect( addPaletteColorShortcut, SIGNAL( activated() ),
```

Comment:
I didn't know one could use slots with default arguments, but yes: https://doc.qt.io/qt-5/signalsandslots.html#signals-and-slots-with-default-arguments